Formula/amber.rb

```diff
@@ -0,0 +1,18 @@
+class Amber < Formula
+  desc "CLI client for generating, scaffolding Amber web applications"
+  homepage "https://www.amberframework.org"
+  url "https://github.com/amberframework/amber/archive/v0.5.0.tar.gz"
+  sha256 "dfcf79a51a5ed6f1693cba0fc4761b4cd4cb7472f35df23182e68e4f4fa29e96"
+
+  depends_on "crystal-lang"
+
+  def install
+    system "shards", "install"
+    system "make", "install", "PREFIX=#{prefix}"
+    bin.install "bin/amber"
+  end
+
+  test do
+    assert_match "Amber CLI (amberframework.org) - v#{version}", shell_output("#{bin}/amber -v")
```

Comment:
Any chance of a test that does more than check the version here?

Reply:
This is sufficient, this CLI scaffolds a project and testing with brew can be cumbersome, which I attempt to do but find it difficult to test with brew

Reply:
What about just doing something like `amber new` and then checking the directory structure is as-expected?

Reply:
I'm going to take a shot at this. :)